### ISSUE TITLE
Only first active profile is used

### ISSUE THREAD

**Issue by nickdbush** [GIScience/openrouteservice#855]
#### Here's what I did
Self-hosted Docker installation, tagged at `v6.3.3` from Docker Hub.
`app.config`: https://gist.github.com/nickdbush/5869f73f7a24460e43a34f4bca69b013

There are multiple active profiles. However, ORS only seems to generate graphs for the first profile in the list.

Using elevation workaround (`gmted` instead of `multi`), elevation disabled for each profile.

`BUILD_GRAPHS` set to `True` to ensure reconstruction when config file changed.

---
#### Here's what I got
When `cycling-regular` is the first active profile in the array, the only line printed out which is relevant is:
`INFO [routing.RoutingProfile] - [1] Profiles: 'cycling-regular', location: 'data/graphs/bike-regular'.`

When `driving-car` is the first profile in the array, it only prints that profile. It generates the graph for the first profile, and then stops and appears in the "ready" state when health-checked.

---
#### Here's what I was expecting
For graphs to be created for each profile. When requesting directions, it is only possible to use the first active profile, the others come back saying that the RoutingProfile could not be found.

**Comment by nilsnolde** [comment]
What were the exact steps you did to get there? What's your container configuration/docker-compos.yml (volumes, env vars)?

E.g.
```
docker pull openrouteservice/openrouteservice
docker run -dt --name ors -v $PWD/conf:/ors-conf ...
# edit ./conf/app.config
docker restart ors
```

It should work in any case when you restart an existing container with the edited app.config. It _might_ not work if you actually rebuilt the image or start a brand new fresh container with an existing app.config. I think the latter is an undefined case in the `docker_entrypoint.sh` script:
https://github.com/GIScience/openrouteservice/blob/8bf5d95cc730cb1f96243d2541d586ede2a57b4c/docker-entrypoint.sh#L22-L32

No behavior is specified for handling the `app.config` when it's not present but the container is started for the first time (what it should do then is `cp -f /ors-conf/app.config $source_appconfig` before tomcat builds the app the first time). Which should result in the default being used. That can't be your problem actually, since the default is always `car` as defined in the Dockerfile:
https://github.com/GIScience/openrouteservice/blob/8bf5d95cc730cb1f96243d2541d586ede2a57b4c/Dockerfile#L32-L33

**Comment by nilsnolde** [comment]
BTW, the workaround with elevation provider shouldn't be necessary anymore. Solving that was the only point of the new v6.3.3 release.

**Comment by takb** [comment]
fixed by #856
